ms-campus-kid-grades/src/db_postgres/app.py

```diff
@@ -0,0 +1,101 @@
+import json
+from flask import Flask
+from flask import request
+import flask_sqlalchemy
+import os
+
+#user = os.environ['postgres']
```

Comment:
Why do we need these comments?

Reply:
I don't need it.

Reply:
In that case, please delete all of them.